dipy/align/tests/test_imaffine.py

```diff
@@ -186,23 +186,26 @@ def test_affreg_all_transforms():
     # Test affine registration using all transforms with typical settings
 
     # Make sure dictionary entries are processed in the same order regardless
-    # of the platform.
-    # Otherwise any random numbers drawn within the loop would make
-    # the test non-deterministic even if we fix the seed before the loop.
-    # Right now, this test does not draw any samples,
-    # but we still sort the entries
-    # to prevent future related failures.
+    # of the platform. Otherwise any random numbers drawn within the loop would
+    # make the test non-deterministic even if we fix the seed before the loop.
+    # Right now, this test does not draw any samples, but we still sort the
+    # entries to prevent future related failures.
     for ttype in sorted(factors):
         dim = ttype[1]
         if dim == 2:
             nslices = 1
         else:
-            nslices = 45
+            nslices = 20
```

Comment:
Hello @skoudoro, I see you have reduced the number of slices. I suppose this is for speeding up the tests? Maybe this will create problems in the future. Let's have an eye for this.

Reply:
I think we can put it back as it was. This was not the main reason of the infinite build. It should ok now. Can you do that @arokem please? Thank you